### PR TITLE
fix(i18n): #932

### PR DESCRIPTION
**Pull request by valentinoli**

### CODE REVIEW

File: experiences/components/chart/view/ChartViewOverviewUber.vue [hunk 1]
```diff
@@ -38,35 +38,35 @@
...
                   <span class="filter" />
                 </span>
               </p>
             </div>
           </VCol>
           <VCol cols="12" md="4">
             <div id="week-chart">
-              <strong>Day of week</strong>
+              <strong v-t="k('Day of week')" />
               <a class="reset" style="display: none">reset</a>
```
Comment (fquellec) [inline experiences/components/chart/view/ChartViewOverviewUber.vue:54]: We have a translation key for reset too: 
`<a class="reset" style="display: none" v-t="'reset'"></a>`

**Review by fquellec — CHANGES_REQUESTED** [review]
Nice way of handling days !

**Comment by valentinoli** [comment]
Commit 7d35e92 should have the following commit message: `fix: use $n() method options to round numbers`

**Review by valentinoli — COMMENTED** [review]
Approve

**Pull request merged**